composer.json

```diff
@@ -6,7 +6,7 @@
     "license": "MIT",
     "require": {
         "php": "^7.4",
-        "sylius/sylius": ">=1.7.9 <1.8.0"
```

Comment:
Please leave Sylius 1.8 out of this. It is not related to the Spanish translation. There already is a PR for that, but currently the tests are broken for 1.7 and that needs to be fixed first.

Reply:
Sorry, this change was for my local branch i will remove it.